src/main/java/io/ticktok/validator/ClockRequest.java

```diff
@@ -0,0 +1,40 @@
+package io.ticktok.validator;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class ClockRequest {
+
+    private String name;
+    private String schedule;
+
+    private ClockRequest(String name, String schedule) {
+        this.name = name;
+        this.schedule = schedule;
+    }
+
+    public static ClockRequest create(String name, String schedule) {
+        validateRequestParmValid(name);
+        validateRequestParmValid(schedule);
+        return new ClockRequest(name, schedule);
+    }
+
+    private static void validateRequestParmValid(String prop) {
+        if(StringUtils.isBlank(prop)){
+            throw new TicktokInvalidValueException("value cannot be empty");
```

Comment:
general message, so I get an exception but what I did wrong?

Reply:
value is empty. this cannot be clear more then that

Reply:
which value? schedule or clock? 
and it's weird since schedule can be a real value but still invalid

Reply:
fixed